### ISSUE TITLE
Feature: implement a reliable method to kill apps spawned by a specific skywire-node

### ISSUE THREAD

**Issue by ayuryshev** [skycoin/skywire-testnet#388]
**Feature description**

Create a reliable method to kill apps spawned by specific skywire-node

**Is your feature request related to a problem? Please describe.**

Concider the following scenario:
- 3 nodes are running 
- NodeA and NodeC both spawns skychat
- NodeB is intermediary

```bash 
$ ps aux |grep "[N]ode"                                                                                                                             
# isk      10310  6.3  0.4 1451492 33184 pts/7   Sl+  14:03   0:01 ./bin/skywire-node ./integration/messaging/nodeA.json --tag NodeA
# isk      10476 10.7  0.4 1377528 35620 pts/8   Sl+  14:03   0:01 ./bin/skywire-node ./integration/intermediary-nodeB.json --tag NodeB
# isk      10634  7.2  0.4 1377188 33424 pts/9   Sl+  14:03   0:01 ./bin/skywire-node ./integration/messaging/nodeC.json --tag NodeC
$ ps aux |grep "[s]kychat"                                                                                                                          
isk      10444  1.3  0.1 839896 15212 pts/7    Sl+  14:03   0:00 /home/isk/github.com/skycoin/src/skywire-services/bin/apps/skychat.v1.0
isk      10710  1.5  0.1 838492 15492 pts/9    Sl+  14:03   0:00 /home/isk/github.com/skycoin/src/skywire-services/bin/apps/skychat.v1.0 -addr :8001
```
Which one skychat is spawned by which node?

This problem is especially important for implementing integration tests with restart cycles: hanged apps makes impossible to start them reliably.

**Describe the solution you'd like**

Solution that allows to reliably kill apps spawned by specific node without killing apps spawned by the other nodes.

**Describe alternatives you've considered**

- Improve skywire-node app-management in such way that it kills it's apps during startup 
- Improve apps: add some kind of health-check with heartbeats so that apps automatically exits when skywire-node not exists 

**Possible implementation**

1. Use `--tag` param of skywire-node for apps too 

This way it will be possible to distinguish spawned apps with `ps aux|grep "[t]agName"`

2. Use of pid-files

Propose such solution in comments please

**Comment by evanlinjin** [comment]
I've made this clear, but I would choose option: `2. Use of pid-files`. Unless if @ayuryshev has some better ideas?

**Comment by ayuryshev** [comment]
Solution with pid-files residing in local directory of nodes - I agree, it will work.

Small note: it does not conforms with [Unix Filesystem Hierarchy Standard]( http://www.pathname.com/fhs/2.2/fhs-5.13.html)

**Comment by jdknives** [comment]
Closed by  #403
